.github/workflows/CI.yml

```diff
@@ -155,6 +161,7 @@ jobs:
 
       - name: Test
         run: yarn test:ember --launch ${{ matrix.browser }}
+        working-directory: test-app
```

Comment:
@linainu let's do the same adjustment for `required-try-scenarios` and `optional-try-scenarios` jobs